media-plugins/gst-plugins-cdparanoia/files/gst-plugins-base-1.18.4-use-system-libs.patch

```diff
@@ -0,0 +1,58 @@
+commit 7c4351a3ee14c75961481895c0e13bb8672181c5
+Author: Igor V. Kovalenko <igor.v.kovalenko@gmail.com>
+Date:   Sat Jul 3 12:48:39 2021 +0300
+
+    use system gstreamer libs
+
+diff --git a/ext/meson.build b/ext/meson.build
+index d984b5087..178b5169b 100644
+--- a/ext/meson.build
++++ b/ext/meson.build
+@@ -1,6 +1,6 @@
+ subdir('alsa')
+ subdir('cdparanoia')
+-subdir('gl')
++#subdir('gl')
```

Comment:
From the [devmanual](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#how):
>Removed lines should not appear in the patch because they are commented — just remove them entirely. Patches show removed lines by prefixing them with a -, so no information is lost by simply deleting lines rather than commenting them out (which adds noise). This makes the patch shorter and more readable.

I assume this is Gentoo specific and can't be upstreamed too? May be worth putting a bit more info at the top of the patches.